### DISCUSSION TITLE
Does this scan buildSrc?

### DISCUSSION THREAD

**Discussion by piyush-kukadiya** [ben-manes/gradle-versions-plugin#525] [Q&A]
I have declared dependencies in `buildSrc` folder but this plugin not executing `dependencyUpdates` task for it.

**Comment by ben-manes** [comment]
It’s not exposed in the APIs so we cannot inspect it directly. https://github.com/ben-manes/gradle-versions-plugin/issues/284 describes a workaround but I haven’t tried it
